spec/factories/miq_group.rb

```diff
@@ -1,7 +1,8 @@
 FactoryGirl.define do
   factory :miq_group do
+    guid { MiqUUID.new_guid }
+
     sequence(:description) { |n| "Test Group #{seq_padded_for_sorting(n)}" }
-    sequence(:guid) { |n| UUIDTools::UUID.parse_int((0x87654321 << 96) + n) }
```

Comment:
The previous slightly-opaque spelling produces UUIDs of the form `87654321-0000-0000-0000-000000000001`, `87654321-0000-0000-0000-000000000002`, etc.

I wanted something that was at-a-glance obviously artificial -- hence many zeros, and an incrementing number. Then I added the `87654321` segment to make it greppable.

So, while I slightly prefer the idea of recognising a factory value, and thus not confusing it with a "real" UUID that might have some actual meaning... I have no particular attachment to the current implementation.